package.json

```diff
@@ -14,8 +14,9 @@
     "@uiw/codemirror-theme-github": "^4.21.25",
     "@uiw/react-codemirror": "^4.21.25",
     "@uiw/react-textarea-code-editor": "^3.0.2",
-    "antd": ">4, <5",
+    "antd": "^4.24.16",
     "axios": "^1.6.8",
+    "black": "^0.3.0",
```

Comment:
```suggestion
```

Reply:
This package is different from the Python `black` linting package and should be uninstalled via `npm uninstall black` before recommitting